src/lib/routes/admin-api/user-admin.ts

```diff
@@ -123,6 +137,50 @@ export default class UserAdminController extends Controller {
             ],
         });
 
+        this.route({
+            method: 'get',
+            path: '/:id/pat',
+            handler: this.getUserPats,
+            permission: READ_USER_PAT,
+            middleware: [
+                openApiService.validPath({
+                    tags: ['Users'],
```

Comment:
I do not think we should use this tag. Should start with the experimental tag and be "Admin" eventually.